housekeeping/src/main/java/com/hotels/housekeeping/service/impl/FileSystemHousekeepingService.java

```diff
@@ -85,7 +81,7 @@ private void housekeepPath(LegacyReplicaPath cleanUpPath) {
         LOG.warn("Path '{}' does not exist.", cleanUpPath);
       }
       rootPath = deleteParents(fs, path, cleanUpPath.getPathEventId());
-    } catch (Exception e) {
```

Comment:
I took the liberty of changin the exception handling a bit I think it was too lenient and there was the danger of hiding bugs. I understand this leaves us a bit open to potential bugs we didn't see before but I think it is better in the long run.

Reply:
Yeah, I think this is better, worst case we get some kind of exception and it crashes earlier than it would have but then hopefully we'll have a clear stack trace and can quickly fix it.